Dockerfile

```diff
@@ -1,18 +1,17 @@
-FROM phusion/baseimage:latest
-MAINTAINER Jochen Breuer "arpitbatra123@gmail.com"
+FROM phusion/baseimage:0.9.22
+MAINTAINER Jochen Breuer "brejoc@gmail.com"
 RUN apt-get -y update
 RUN add-apt-repository -y ppa:ondrej/php
 RUN apt-get update  
 RUN apt-get install -y --force-yes apache2 mariadb-server libapache2-mod-php
 RUN apt-get install -y  --force-yes php7.0-gd php7.0-json php7.0-mysql php7.0-curl php7.0-mbstring
 RUN apt-get install -y  --force-yes php7.0-intl php7.0-mcrypt php-imagick php7.0-xml php7.0-zip
-RUN apt-get install -y wget unzip
+RUN apt-get install -y php-mysql php-zip php-gd php-json php-curl php-mbstring php-intl php-mcrypt php-imagick php-xml 
 RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 RUN rm -rf /etc/service/sshd /etc/my_init.d/00_regen_ssh_host_keys.sh
 
-RUN wget https://download.nextcloud.com/server/releases/nextcloud-12.0.3.zip
-RUN unzip nextcloud-12.0.3.zip -d /var/www 
+RUN curl -k https://download.nextcloud.com/server/releases/nextcloud-12.0.3.tar.bz2 | tar jx -C /var/www/ 
```

Comment:
This is not working, because we are missing the package `bzip2` to actually unpack the archive. You should add that somewhere above.